rules-tests/TypeDeclaration/Rector/Class_/TypedPropertyFromCreateMockAssignRector/Fixture/skip_value_resetted.php.inc

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Class_\TypedPropertyFromCreateMockAssignRector\Fixture;
+
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Rector\Tests\TypeDeclaration\Rector\Class_\TypedPropertyFromCreateMockAssignRector\Source\SomeMockedClass;
+
+class SkipValueResetted extends TestCase
+{
+    public $someMock;
+
+    protected function setUp(): void
+    {
+        $this->someMock = $this->createMock(SomeMockedClass::class);
+    }
+
+    protected function tearDown(): void
+    {
+        $this->someMock = null;
+    }
+}
```

Comment:
This should be set to `?MockObject` instead

Reply:
sure https://github.com/rectorphp/rector-src/pull/6179